Tests are failing.

I'm using `go version go1.6 darwin/amd64`.

The tests in this package are currently failing:

```
$ go test -v github.com/gregjones/httpcache
=== RUN   Test

----------------------------------------------------------------------
FAIL: httpcache_test.go:564: S.TestStaleIfErrorRequest

httpcache_test.go:590:
    c.Assert(err, Equals, nil)
... obtained *errors.errorString = &errors.errorString{s:"some error"} ("some error")
... expected = nil


----------------------------------------------------------------------
FAIL: httpcache_test.go:594: S.TestStaleIfErrorRequestLifetime

httpcache_test.go:620:
    c.Assert(err, Equals, nil)
... obtained *errors.errorString = &errors.errorString{s:"some error"} ("some error")
... expected = nil


----------------------------------------------------------------------
FAIL: httpcache_test.go:636: S.TestStaleIfErrorResponse

httpcache_test.go:661:
    c.Assert(err, Equals, nil)
... obtained *errors.errorString = &errors.errorString{s:"some error"} ("some error")
... expected = nil


----------------------------------------------------------------------
FAIL: httpcache_test.go:665: S.TestStaleIfErrorResponseLifetime

httpcache_test.go:690:
    c.Assert(err, Equals, nil)
... obtained *errors.errorString = &errors.errorString{s:"some error"} ("some error")
... expected = nil

OOPS: 24 passed, 4 FAILED
--- FAIL: Test (0.01s)
FAIL
exit status 1
FAIL	github.com/gregjones/httpcache	0.023s
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,6 +11,6 @@ install:
   - # Do nothing. This is needed to prevent default install action "go get -t -v ./..." from happening here (we want it to happen inside script step).
 script:
   - go get -t -v ./...
-  - diff -u <(echo -n) <(gofmt -d -s .)
+  - diff -u <(echo -n) <(gofmt -d .)
   - go tool vet .
   - go test -v -race ./...

--- a/httpcache_test.go
+++ b/httpcache_test.go
@@ -565,6 +565,8 @@ func (s *S) TestStaleIfErrorRequest(c *C) {
 	now := time.Now()
 	tmock := transportMock{
 		response: &http.Response{
+			Status:     http.StatusText(http.StatusOK),
+			StatusCode: http.StatusOK,
 			Header: http.Header{
 				"Date":          []string{now.Format(time.RFC1123)},
 				"Cache-Control": []string{"no-cache"},
@@ -595,6 +597,8 @@ func (s *S) TestStaleIfErrorRequestLifetime(c *C) {
 	now := time.Now()
 	tmock := transportMock{
 		response: &http.Response{
+			Status:     http.StatusText(http.StatusOK),
+			StatusCode: http.StatusOK,
 			Header: http.Header{
 				"Date":          []string{now.Format(time.RFC1123)},
 				"Cache-Control": []string{"no-cache"},
@@ -637,6 +641,8 @@ func (s *S) TestStaleIfErrorResponse(c *C) {
 	now := time.Now()
 	tmock := transportMock{
 		response: &http.Response{
+			Status:     http.StatusText(http.StatusOK),
+			StatusCode: http.StatusOK,
 			Header: http.Header{
 				"Date":          []string{now.Format(time.RFC1123)},
 				"Cache-Control": []string{"no-cache, stale-if-error"},
@@ -666,6 +672,8 @@ func (s *S) TestStaleIfErrorResponseLifetime(c *C) {
 	now := time.Now()
 	tmock := transportMock{
 		response: &http.Response{
+			Status:     http.StatusText(http.StatusOK),
+			StatusCode: http.StatusOK,
 			Header: http.Header{
 				"Date":          []string{now.Format(time.RFC1123)},
 				"Cache-Control": []string{"no-cache, stale-if-error=100"},

